plugins/kuadrant/src/components/EditAPIKeyRequestDialog/index.ts

```diff
@@ -1 +1 @@
-export { EditAPIKeyRequestDialog } from './EditAPIKeyRequestDialog';
+export { EditAPIKeyRequestDialog } from './EditAPIKeyRequestDialog.tsx';
```

Comment:
this is unexpected... the other components index do not need to specify .tsx extension.

Reply:
mhmm, most probably my IDE messing it up